package.json

```diff
@@ -59,7 +59,7 @@
   "dependencies": {
     "axios": "0.19.0",
     "debug": "4.1.1",
-    "handlebars": "4.4.2",
+    "handlebars": "4.5.1",
```

Comment:
Has to be 4.5.3 to solve all vulnerabilities if I believe NPM audit :
- Prototype pollution (High)
- Arbitrary code execution (High) => Two of them